src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -182,12 +183,10 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
     []
   );
 
-  if (!venueId || !venue) return null;
-
-  const isVenueUsingPartyMap = venue.template === VenueTemplate.partymap;
+  const isVenueUsingPartyMap = venue?.template === VenueTemplate.partymap;
 
   // TODO: ideally this would find the top most parent of parents and use those details
-  const navbarTitle = parentVenue?.name ?? venue.name;
+  const navbarTitle = parentVenue?.name ?? venue?.name;
```

Comment:
Should we also have a `??` default for when `venue?.name` is undefined? This will be when there is no venue present at all yeah?

Reply:
What would that default value be? We don't have a default venue. If this is undefined it should probably redirect to somewhere else.

Reply:
Unsure, I didn’t deeply trace the logic of this so don’t know how it’s used/intended; just saw that it will can now end up undefined, whereas before it at least claimed that it couldn’t be.

Reply:
@denisdimitrov This is still open.. can you please provide an answer one way or the other based on my comments above.